sceptre/cli/list.py

```diff
@@ -79,6 +80,9 @@ def list_outputs(ctx, path, export):
     plan = SceptrePlan(context)
     responses = [response for response in plan.describe_outputs().values() if response]
 
+    # Legacy. This option was added in the initial commit of the project,
+    # although its intended use case is unclear. It may relate to a feature
+    # that had been removed prior to the initial commit.
```

Comment:
i believe this is intended to be used with the [sceptre environment variable resolver](https://docs.sceptre-project.org/4.2.2/docs/resolvers.html#environment-variable).

Reply:
Ok, noted. Will amend the comment.

Reply:
Because I haven't done anything yet!

Reply:
@zaro0508 I am actually looking at this now. I still can't quite see the connection. How does code that exports env vars as `SCEPTRE_some_var` connect to the `!environment_variable` resolver? I would think the resolver simply allows us to use environment variables as inputs, but nothing specifically to do with environment variables whose names are prefixed with `SCEPTRE_`?

The feature reminds me of the `TF_VAR_some_var` feature that Terraform provides. But as far I can tell, Sceptre hasn't implemented anything similar.

Reply:
Is this point resolved @zaro0508 i.e. we really don't know what the `SCEPTRE_env_vars` are for and no reason for a comment in the code not to say so? Otherwise, happy to change or remove the comment as you see fit.

Reply:
yes, i'm fine to keep as is